examples/imagenet/create_data.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+../../build/bin/createdata -trainlist "/data/xiangrui/label/train.txt" -trainfolder "/data/xiangrui/ILSVRC2012_img_train" \
```

Comment:
pls remove the path.
you can add a readme file to include the running instructions.